qutebrowser/commands/command.py

```diff
@@ -101,10 +100,14 @@ def __init__(self, *, handler, name, instance=None, maxsplit=None,
             for m in modes:
                 if not isinstance(m, usertypes.KeyMode):
                     raise TypeError("Mode {} is no KeyMode member!".format(m))
-        if not_modes is not None:
+            self._modes = modes
+        elif not_modes is not None:
             for m in not_modes:
                 if not isinstance(m, usertypes.KeyMode):
                     raise TypeError("Mode {} is no KeyMode member!".format(m))
+            self._modes = [m for m in usertypes.KeyMode if m not in not_modes]
```

Comment:
Why not make `self._modes` a set? Then we could simply do `set(usertypes.KeyMode).difference(not_modes)` if I'm not mistaken.